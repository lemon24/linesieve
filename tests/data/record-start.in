--section '\S+:$' --record-start '^\d+:\d+ '
match -s before-and-one-line one
match -s multi-line three
---
before-and-one-line:
zero
one
00:01 one
00:02 two
three
00:03 three
3
multi-line:
zero
one
00:01 one
00:02 two
three
00:03 three
3
