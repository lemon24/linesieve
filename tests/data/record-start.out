before-and-one-line:
zero
one
00:01 one
multi-line:
00:02 two
three
00:03 three
3
